__tests__/app.test.js

```diff
@@ -314,5 +314,57 @@ describe('6. POST /api/articles/:article_id/comments', () => {
 })
 
 describe('7. PATCH /api/articles/:article_id', () => {
+    describe('a. status 200 & data', () => {
+        it('returns 200 status code & the article with the updated vote', () => {
+            const inc_votes = {inc_votes : 5}
+
+            const result = {
+                article_id: 11,
+                title: "Am I a cat?",
+                topic: "mitch",
+                author: "icellusedkars",
+                body: "Having run out of ideas for articles, I am staring at the wall blankly, like a cat. Does this make me a cat?",
+                created_at: "2020-01-15T22:21:00.000Z",
+                votes: 5,
+              }
+
+            return request(app)
+            .patch('/api/articles/11')
+            .send(inc_votes)
+            .expect(202)
+            .then ( (res) => {
+                const {article} = res.body;
+                expect(article).toEqual(result)

```

Comment:
Would still suggest using `expect.objectContaining` or `toMatchObject` here instead of `toEqual` alone